src/Form/AdminSettingsForm.php

```diff
@@ -68,7 +61,7 @@ public static function create(ContainerInterface $container) {
   /**
    * {@inheritdoc}
    */
-  public function buildForm(array $form, FormStateInterface $form_state, Request $request = NULL) {
+  public function buildForm(array $form, FormStateInterface $form_state) {
```

Comment:
I have created LEX-986 to capture the dropdown item, along with the rest of items that are valuable but I did not get to.